docs/index.html

```diff
@@ -155,7 +155,15 @@ <h2>What is the purpose?</h2>
           quadrant: 2,
           ring: 1,
           label: "Redis",
-          website: "",
+          website: "https://redis.io/",
+          description: `
+          <p>Redis is an open source in-memory data store, used as a database, cache and message broker.</p>
+          <p>
+            In Small Improvements we use Google Cloud Memory Store Redis as a cache.
+          Google App Engine recommends using the Cloud Memory Store <a href="https://cloud.google.com/appengine/docs/standard/java11/java-differences#memcache"> as a migration path from the memcache originally built into App Engine</a>. 
```

Comment:
I'd always open external links in a new tab. It's a bit of boilerplate but generates a better UX, imo – what do you think?
```suggestion
          Google App Engine recommends using the Cloud Memory Store <a href="https://cloud.google.com/appengine/docs/standard/java11/java-differences#memcache" target="_blank" rel="noopener noreferrer"> as a migration path from the memcache originally built into App Engine</a>. 
```